frontend/src/store.js

```diff
@@ -0,0 +1,29 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
+
+export default new Vuex.Store({
+  state: {
+    students: []
+  },
+  mutations: {
+    SET_STUDENTS(state, students) {
+      state.students = students
+    }
+  },
+  actions: {
+    getAllStudents( {commit} ) {
+      return fetch('http://localhost:3001/api/students')
```

Comment:
you don't need to have return, it will call the commit when have a data
what do you think if we can use async/await?

Reply:
oh i see! Thanks for the advice, have removed the `return` and used async/await